src/include/planner/plannodes/hash_plan_node.h

```diff
@@ -81,7 +81,7 @@ class HashPlanNode : public AbstractPlanNode {
   /**
    * @return keys to be hashed on
    */
-  const std::vector<std::shared_ptr<parser::AbstractExpression>> &GetHashKeys() const { return hash_keys_; }
+  const std::vector<std::unique_ptr<parser::AbstractExpression>> &GetHashKeys() const { return hash_keys_; }
```

Comment:
I think this should be `std::vector<std::ManagedPointer<....>>`

Reply:
Yup